### PR TITLE
Enterprise docker getting started fixes

### PR DESCRIPTION
**Pull request by stevenGravy**
License copying missing for docker-compose
Use license.pem filename for docker example
use branch instead of master for file references.

### CODE REVIEW

File: docs/pages/enterprise/getting-started.mdx [hunk 1]
```diff
@@ -78,7 +78,7 @@ and save it as `/var/lib/teleport/license.pem` on the auth server.
 
 ### Systemd Unit File
 
-Next, download the systemd service unit file from [examples directory](https://github.com/gravitational/teleport/tree/master/examples/systemd)
+Next, download the systemd service unit file from [examples directory](https://github.com/gravitational/teleport/tree/(=teleport.version=)/examples/systemd)
```
Comment (ptgott) [inline docs/pages/enterprise/getting-started.mdx:81]:
```suggestion
Download the systemd unit file for Teleport from the [examples directory](https://github.com/gravitational/teleport/tree/v(=teleport.version=)/examples/systemd)
```

File: docs/pages/enterprise/getting-started.mdx [hunk 2]
```diff
@@ -366,11 +366,15 @@ We currently only offer Docker images for `x86_64` architectures.
   You will need a recent version of [`docker-compose`](https://docs.docker.com/compose/install/) installed to follow this section of the quick start guide.
 </Admonition>
 
-The easiest way to start Teleport Enterprise quickly is to use `docker-compose` with our [`teleport-ent-quickstart.yml`](https://github.com/gravitational/teleport/blob/master/docker/teleport-ent-quickstart.yml) file:
+The easiest way to start Teleport Enterprise quickly is to use `docker-compose` with our [`teleport-ent-quickstart.yml`](https://github.com/gravitational/teleport/blob/(=teleport.version=)/docker/teleport-ent-quickstart.yml) file:
```
Comment (ptgott) [inline docs/pages/enterprise/getting-started.mdx:369]:
```suggestion
The easiest way to start Teleport Enterprise quickly is to use `docker-compose` with our [`teleport-ent-quickstart.yml`](https://github.com/gravitational/teleport/blob/v(=teleport.version=)/docker/teleport-ent-quickstart.yml) file:
```

File: docs/pages/enterprise/getting-started.mdx [hunk 3]
```diff
@@ -366,11 +366,15 @@ We currently only offer Docker images for `x86_64` architectures.
...
 
 ```code
+# download your license file from the Gravitational dashboard and put it in the correct directory
```
Comment (ptgott) [inline docs/pages/enterprise/getting-started.mdx:372]:
```suggestion
# Download your license file from the Gravitational dashboard and put it in the correct directory.
```

File: docs/pages/enterprise/getting-started.mdx [hunk 4]
```diff
@@ -366,11 +366,15 @@ We currently only offer Docker images for `x86_64` architectures.
...
+# the file needs to be named license.pem
```
Comment (ptgott) [inline docs/pages/enterprise/getting-started.mdx:373]:
```suggestion
# The file needs to be named "license.pem".
```

File: docs/pages/enterprise/getting-started.mdx [hunk 5]
```diff
@@ -366,11 +366,15 @@ We currently only offer Docker images for `x86_64` architectures.
...
+$ cp ~/downloads/license.pem .
+
 # download the quickstart file from our Github repo
-$ curl -Lso teleport-ent-quickstart.yml https://raw.githubusercontent.com/gravitational/teleport/master/docker/teleport-ent-quickstart.yml
+$ curl -Lso teleport-ent-quickstart.yml https://raw.githubusercontent.com/gravitational/teleport/(=teleport.version=)/docker/teleport-ent-quickstart.yml
```
Comment (ptgott) [inline docs/pages/enterprise/getting-started.mdx:377]:
```suggestion
$ curl -Lso teleport-ent-quickstart.yml https://raw.githubusercontent.com/gravitational/teleport/v(=teleport.version=)/docker/teleport-ent-quickstart.yml
```

**Comment by zmb3** [comment]
@stevenGravy are you still looking to get this merged or should we close it out?

**Comment by stevenGravy** [comment]
> @stevenGravy are you still looking to get this merged or should we close it out?

Yes still looking, @zmb3 , I've applied @ptgott review edits. @ptgott can you confirm?

File: docs/pages/enterprise/getting-started.mdx [hunk 6]
```diff
@@ -91,8 +91,8 @@ and save it as `/var/lib/teleport/license.pem` on the auth server.
 
 ### Systemd Unit File
 
-Next, download the systemd service unit file from [examples directory](https://github.com/gravitational/teleport/tree/master/examples/systemd)
-on GitHub and save it as `/etc/systemd/system/teleport.service` on both servers.
+Download the systemd unit file for Teleport from the [examples directory](https://github.com/gravitational/teleport/tree/v(=teleport.version=)/examples/systemd)
+on Github and save it as `/etc/systemd/system/teleport.service` on both servers.
```
Comment (ptgott) [inline docs/pages/enterprise/getting-started.mdx:95]: Not sure why this change is being made

File: docs/pages/enterprise/getting-started.mdx [hunk 7]
```diff
@@ -379,11 +379,15 @@ We currently only offer Docker images for `x86_64` architectures.
...
+The easiest way to start Teleport Enterprise quickly is to use `docker-compose` with our [`teleport-ent-quickstart.yml`](https://github.com/gravitational/teleport/blob/v(=teleport.version=)/docker/teleport-ent-quickstart.yml) file:
 
 ```code
-# download the quickstart file from our GitHub repo
-$ curl -Lso teleport-ent-quickstart.yml https://raw.githubusercontent.com/gravitational/teleport/master/docker/teleport-ent-quickstart.yml
+# Download your license file from the Gravitational dashboard and put it in the correct directory.
+# The file needs to be named "license.pem".
+$ cp ~/downloads/license.pem .
+
+# Download the quickstart file from our Github repo
```
Comment (ptgott) [inline docs/pages/enterprise/getting-started.mdx:389]:
```suggestion
# Download the quickstart file from our GitHub repo
```

File: docs/pages/enterprise/getting-started.mdx (hunk 6 above)
Comment (stevenGravy) [reply docs/pages/enterprise/getting-started.mdx]: fixed the camelcase of GitHub @ptgott

**Review by ptgott — APPROVED** [review]

**Review by zmb3 — APPROVED** [review]
